xml/decoders/SoundTraxx_Tsu2_Steam_OEM_Genesis.xml

```diff
@@ -42,15 +42,55 @@
             <date>2021-01-21</date>
             <authorinitials>MM</authorinitials>
             <revremark>Update</revremark>
+        </revision>
+		<revision>
```

Comment:
This line has TAB characters instead of spaces in the beginning of the line. It would be good to change these TABs to spaces.

Reply:
Corrected, replaced  TAB characters  with spaces as found